.travis/setup-build.sh

```diff
@@ -4,9 +4,30 @@ set -o errexit
 
 source "$( dirname "${BASH_SOURCE[0]}" )/travis_retry.sh"
 
+cd "$TRAVIS_BUILD_DIR/build"
+
+case "$TRAVIS_EVENT_TYPE" in
+    push)
+        echo "Checking out branch '$TRAVIS_BRANCH'"
+        git checkout -qf "$TRAVIS_BRANCH"
+        echo 'Installing Node packages as per package-lock.json file'
+        travis_retry npm ci
+        echo 'Updating bedrock to the latest commit in the default branch'
+        npm install https://github.com/concrete5/bedrock.git
```

Comment:
Can this be `npm update @concretecms/bedrock`?

Reply:
Still curious on this

Reply:
IDK, I guess I used this syntax instead of ssh becausewe may not have loaded any applicable ssh key